acsf_tools.drush.inc

```diff
@@ -434,6 +434,7 @@ function _drush_acsf_tools_prompt_confirm() {
   $yes = drush_get_context('DRUSH_AFFIRMATIVE');
   if ($yes) {
     drush_set_context('DRUSH_AFFIRMATIVE', FALSE);
```

Comment:
I think we should keep confirmation at L440 to prevent an accidental command execution on all the sites of a factory

Reply:
I confirm it was the initial intent but I guess that if you want to script that you need a way to avoid this
A new option called "no-prompt" maybe?

Reply:
:-)